.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    env:
+      TERRAFORM_VERSIONS: "1.0.5 0.14.5 0.13.6 0.12.30 0.11.14"
```

Comment:
I guess there are not too many users with the oldest Terraform versions in the list, but testing with a couple of extra versions is not so heavy (compared to the Ruby versions).